### PR TITLE
Theme thinking

### PR DESCRIPTION
**Pull request by ThalusA**
Please refer to this for in-depth color thinking:

primary:
- light: `#ffffff`,
- main: `#ffffff`,
- dark: `#cccccc`,
- contrastText: `#000000`

secondary:
- light: `#a975ff`,
- main: `#7447d5`,
- dark: `#3d19a3`,
- contrastText: `#ffffff`

background:
- default: `#eee9fa`,
- paper: `#000000`

### CODE REVIEW

**Review by SBasique — APPROVED** [review]

File: utils/Theme.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    light: "#a975ff",
+    main: "#7447d5",
+    dark: "#3d19a3",
+    contrastText: "#ffffff"
+}
+
+export const background: Partial<TypeBackground> = {
+    default: "#eee9fa",
+    paper: "#000000"
+}
```
Comment (NaadiQmmr) [inline utils/Theme.ts:21]: * The width of the rectangle of the input of the search bar is too large while there is no focus
* The police of "Login" is not as bold, and it is in uppercase

**Review by NaadiQmmr — CHANGES_REQUESTED** [review]

File: utils/Theme.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { PaletteColorOptions } from '@mui/material';
+import { TypeBackground } from '@mui/material/styles/createPalette';
+
+export const primary: PaletteColorOptions  = {
+    light: "#ffffff",
+    main: "#ffffff",
+    dark: "#cccccc",
```
Comment (Flavienallix) [inline utils/Theme.ts:7]: When not hovering the button is like that :
image
But when hovering the button is like that :
image

I don't think it makes sense to be like that, so i propose to replace this to make it like that when hovering :
image

```suggestion
    dark: "#3d19a3",
```

File: utils/Theme.ts (hunk 1 above)
Comment (ThalusA) [reply utils/Theme.ts:21]: What do you mean by "The police of "Login" is not as bold, and it is in uppercase"?

File: utils/Theme.ts (hunk 2 above)
Comment (ThalusA) [reply utils/Theme.ts:7]: I proposed a better change :
```tsx
<Button variant="contained" sx={{ color: "primary.main", bgcolor: "secondary.main", '&:hover': { color: "primary.main", bgcolor: 'secondary.dark' }}}>Login</Button>
```

**Review by scriptanium — APPROVED** [review]
Seems good

File: utils/Theme.ts (hunk 1 above)
Comment (NaadiQmmr) [reply utils/Theme.ts:21]: The one in the proto : 
image
The one in the code :
image

Comment (NaadiQmmr) [reply utils/Theme.ts:21]: As you can see, yours is in uppercase, and the police is lighter

Comment (ThalusA) [reply utils/Theme.ts:21]: image
Is it better now ?

**Review by Flavienallix — APPROVED** [review]
**good stuff**

File: utils/Theme.ts (hunk 1 above)
Comment (NaadiQmmr) [reply utils/Theme.ts:21]: Yes !

**Review by NaadiQmmr — APPROVED** [review]

**Pull request merged**